src/main/java/com/example/ludogoriesoft/lukeriaerpapi/dtos/CartonDTO.java

```diff
@@ -17,6 +17,5 @@ public class CartonDTO {
     private String size;
     @Min(value = 1, message = "Наличните бройки трябва да бъдат по-големи от 0!")
     private Integer availableQuantity;
-    @Min(value = 1, message = "Цената трябва да бъде по-голяма от 0!")
-    private double price;
+    private Double price;
```

Comment:
I think that it's a bad practice do use Double for price.
Please, check these articles: 
1. https://medium.com/@cancerian0684/which-data-type-would-you-choose-for-storing-currency-values-like-trading-price-dd7489e7a439
2. https://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency